modules/sdk-coin-ada/src/transaction.ts

```diff
@@ -1,17 +1,185 @@
-import { BaseTransaction, BaseKey, NotImplementedError } from '@bitgo/sdk-core';
+import {
```

Comment:
Let's move all transaction/transfer files to `lib` for consistency with other coins.